frontend/src/widgets/GoalStatusGraph.js

```diff
@@ -174,7 +174,7 @@ export function GoalStatusChart({ data, loading }) {
         )
         : (
           <>
-            <div className="border-top border-gray-5">
+            <div className="border-top border-gray-5" data-testid="goalStatusGraph">
```

Comment:
Always feel like we should use the data-testid more often for ease of use in FE tests.